src/components/Tabs/TabPanel/TabPanel.types.ts

```diff
@@ -0,0 +1,5 @@
+export type TabPanelProps = {
+  children?: React.ReactNode;
```

Comment:
Could you pls import ReactNode type from React on the top?

Reply:
not necessary here